README.md

```diff
@@ -6,6 +6,10 @@
 
 This repository is the source for the Docker Hub image `renovate/buildpack`. Commits to `main` branch are automatically built and published.
 
+## Notice: Do not use the laguage docker tags
+
+Do not use the language docker tags, they are deprecated and won't be updated soon.
```

Comment:
Do you mean docker images like `renovate/ruby` ?

Reply:
```suggestion
Do not use the language docker tags like `renovate/buildpack:ruby` or `renovate/buildpack:5-ruby`, they are deprecated and won't be updated soon.
Use `renovate/buildpack` or `renovate/buildpack:5` images but checkout latest version.
```

Something like this?